package.json

```diff
@@ -391,6 +391,11 @@
 					"type": "boolean",
 					"default": false,
 					"description": "Whether to automatically send a 'hot reload' request during a Flutter debug session when saving files."
+				},
+				"dart.analyzeAngularTemplates": {
```

Comment:
We should probably add this to `getAnalyzerSettings` [here](https://github.com/Dart-Code/Dart-Code/blob/c033ff08ab52cc4da9f67bcb6cb34609db624cd6/src/extension.ts#L368) so that changes to it prompt the user to reload (otherwise the change will appear to not take effect).